Source/AzureSqlBulkFailover.md

```diff
@@ -0,0 +1,191 @@
+# AzureSqlBulkFailover.ps1
+
+This script is used to failover all databases and elastic pools in a subscription to a secondary, already upgraded replica. It filters by subscription, resource group and logical server name. It is intended to be used to facilitate CMW customers to upgrade their databases on demand when upgrades are ready (one touch).
+
+## Usage
+
+This script is intended to be run as an Azure Automation Runbook or locally.
+
+## Parameters
+
+- `SubscriptionId`: The ID of the subscription to failover. Optional.
+- `ResourceGroupName`: The name of the resource group to failover. Optional.
+- `LogicalServerName`: The name of the logical server to failover. Optional.

```

Comment:
I think we should document how RG and LSN combine/interact with each other. 

If one is null and the other is non-null, is only the non-null value used? 

If both are non-null, are they combined with Boolean AND, or OR?